.github/workflows/release.yaml

```diff
@@ -65,10 +84,34 @@ jobs:
               "text": "Build release binary result (${{ matrix.target }}): ${{ job.status }}\n${{ github.event.pull_request.html_url || github.event.head_commit.url }}"
             }
 
-  publish:
-    name: Publish
+
+  docker-push:
+    name: Build and Push Docker Image
+    runs-on: ubuntu-latest-8-cores
+    needs: atomic-build
```

Comment:
This here is called `atomic-build`, but the job at the top is called `atomic-release`.

Reply:
```suggestion
    needs: atomic-release
```